ceph.spec.in

```diff
@@ -271,6 +271,7 @@ BuildRequires:  re2-devel
 %if 0%{with jaeger}
 BuildRequires:  bison
 BuildRequires:  flex
+BuildRequires:  rapidjson-devel
```

Comment:
don't think this is a jaeger dependency, might want to shift it to after pkgconfig(udev)